### ISSUE TITLE
ERROR when analyzing ticker 

### ISSUE THREAD

**Issue by tkso1997** [gcarq/freqtrade#255]
```
2017-12-29 01:37:20,733 - freqtrade.analyze - ERROR - Unexpected error when analyzing ticker for pair BTC_NXT.
Traceback (most recent call last):
  File "/home/thorsten/freqtrade/freqtrade/freqtrade/analyze.py", line 143, in get_signal
    dataframe = analyze_ticker(ticker_hist)
  File "/home/thorsten/freqtrade/freqtrade/freqtrade/analyze.py", line 125, in analyze_ticker
    dataframe = populate_indicators(dataframe)
  File "/home/thorsten/freqtrade/freqtrade/freqtrade/analyze.py", line 44, in populate_indicators
    dataframe['sar'] = ta.SAR(dataframe)
  File "talib/abstract.pyx", line 358, in talib.abstract.Function.__call__ (talib/abstract.c:6913)
  File "talib/abstract.pyx", line 389, in talib.abstract.Function.__call_function (talib/abstract.c:7620)
  File "talib/func.pyx", line 9715, in talib.func.SAR (talib/func.c:102410)
Exception: input lengths are different
```

**Comment by eliashassing** [comment]
I also got this error with a variety of trading pairs.

**Comment by shusso** [comment]
I hit this also. But at least for me the bot continued onwards happily. Will investigate if it happens more

**Comment by tkso1997** [comment]
I get a lot of these now (once every 1-2 minutes), any ideas?


2017-12-29 13:42:36,943 - freqtrade.analyze - WARNING - Unable to analyze ticker for pair BTC_DGB: Shape of passed values is (3, 5759), indices imply (3, 3627)
2017-12-29 13:42:40,469 - freqtrade.analyze - ERROR - Unexpected error when analyzing ticker for pair BTC_ADX.
Traceback (most recent call last):
  File "/home/thorsten/freqtrade/freqtrade-dev/freqtrade/analyze.py", line 137, in get_signal
    dataframe = analyze_ticker(ticker_hist)
  File "/home/thorsten/freqtrade/freqtrade-dev/freqtrade/analyze.py", line 119, in analyze_ticker
    dataframe = populate_indicators(dataframe)
  File "/home/thorsten/freqtrade/freqtrade-dev/freqtrade/analyze.py", line 50, in populate_indicators
    dataframe['mfi'] = ta.MFI(dataframe)
  File "talib/abstract.pyx", line 358, in talib.abstract.Function.__call__ (talib/abstract.c:6913)
  File "talib/abstract.pyx", line 389, in talib.abstract.Function.__call_function (talib/abstract.c:7620)
  File "talib/func.pyx", line 8463, in talib.func.MFI (talib/func.c:90909)
Exception: input lengths are different

**Comment by shusso** [comment]
@tkso1997 most likely found the root cause, there is a #266 open for discussion on what should be the proper way to fix this issue.

**Comment by shusso** [comment]
Should be fixed by revert in #275

**Comment by shusso** [comment]
this is fixed now in develop
